packages/api/src/schema/schema.yaml

```diff
@@ -221,6 +221,9 @@ components:
           type: string
           writeOnly: true
           description: shared secret used to sign the webhook payload
+        streamId:
+          type: string
+          description: streamId of the stream on which the webhook is applied
```

Comment:
Hopefully this field won't need to be indexed right? Raising the discussion here just in case.